cypress/integration/fairOrganizer.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { visitWithStatusRetries } from "../helpers/visitWithStatusRetries"
+
+describe("FairOrganizer", () => {
+  it("/fair-organizer/:slug", () => {
+    visitWithStatusRetries("fair-organizer/expo-chicago-2019")
```

Comment:
I don't think we're fetching any real data at the moment but I think the slug would be `fair-organizer/expo-chicago` for the organizer page. `expo-chicago-2019` refers to the specific fair slug.